### DISCUSSION TITLE
Disable glance

### DISCUSSION THREAD

**Discussion by axelson** [emacs-lsp/lsp-mode#3181] [Q&A] [answered]
In a buffer if I run `lsp-ui-doc-glance` then glance will be disabled. How do I disable it by default (even if the default setting for glance changes in the future)?

**Comment by yyoncho** [comment, marked as answer]
> then glance will be disabled.

Can you elaborate on that?

**Reply by axelson** [reply]
Ah, I guess I actually want to disable `lsp-ui-doc-mode`, looks like glance just triggers the popup in the current buffer.

**Reply by yyoncho** [reply]
To disable lsp-ui-doc use lsp-ui-doc-enable. FTR you may use https://emacs-lsp.github.io/lsp-mode/tutorials/how-to-turn-off/ as a guide.

**Reply by axelson** [reply]
Ah, thanks for the link, that's helpful :+1:

**Comment by ericdallo** [comment]
I opened [this](https://github.com/emacs-lsp/lsp-mode/discussions/3182) as IMO we should not have the glance on cursor hover, but only on mouse hover

**Reply by axelson** [reply]
Ah, that would be great in my opinion :+1:

**Reply by ericdallo** [reply]
Yeah, I noticed that some days ago and just remembered to open the issue right now
